editoast/src/views/path/properties.rs

```diff
@@ -275,16 +208,52 @@ async fn cache_path_properties(
 }
 
 /// Compute path properties input hash without supported electrifications
-fn path_properties_input_hash(
-    infra: i64,
-    infra_version: i64,
-    track_ranges: &[TrackRange],
-) -> String {
+fn path_properties_input_hash(path_properties_request: &PathPropertiesRequest<'_>) -> String {
     let osrd_version = get_app_version().unwrap_or_default();
     let mut hasher = DefaultHasher::new();
-    track_ranges.hash(&mut hasher);
+    path_properties_request
+        .track_section_ranges
+        .hash(&mut hasher);
     let hash_track_ranges = hasher.finish();
-    format!("path_properties.{osrd_version}.{infra}.{infra_version}.{hash_track_ranges}")
+    format!(
+        "path_properties.{osrd_version}.{}.{}.{hash_track_ranges}",
+        path_properties_request.infra, path_properties_request.expected_version
+    )
```

Comment:
nitpick: I feel it's harder to see the different part of the key when some are inlined and other not. See the difference between your version and either of these two versions, which are easier to grasp (at least for me).

```rust
format!("path_properties.{osrd_version}.{infra}.{infra_version}.{hash_track_ranges}")
```
or
```rust
format!("path_properties.{}.{}.{}.{}",
   osrd_version,
   infra,
   infra_version,
   hash_track_ranges,
)
```

If you'd prefer to keep some of them inlined, you could do something like this I believe.

```suggestion
    format!(
        "path_properties.{osrd_version}.{infra}.{infra_version}.{hash_track_ranges}",
        infra = path_properties_request.infra,
        infra_version = path_properties_request.expected_version
    )
```

Reply:
Done.